library/swarm

```diff
@@ -0,0 +1,4 @@
+# maintainer: Andrea Luzzardi <al@docker.com> (@aluzzardi)
+# maintainer: Victor Vieux <vieux@docker.com> (@vieux)
+
+latest: git://github.com/docker/swarm-library-image@c0f8204b2a1bb34233cd2144df51a4de9572440c
```

Comment:
We are planning to keep older versions - `rc`s are kinda different in the sense that they are not really releases.

We could go either way - I'm not entirely sure.

We're not planning to push rc's in the library (at least, we don't think so unless we find a compelling reason).